tests/kernel/udev_no_symlink.pm

```diff
@@ -49,7 +50,8 @@ sub run {
     assert_script_run('journalctl -u detect-part-label-duplicates.service --no-pager | grep "Warning: a high number of partitions uses"');
     assert_script_run('test $(grep -r "E:ID_PART_ENTRY_NAME=primary" /run/udev/data | wc -l) -eq ' . $num_primary);
     assert_script_run('test $(grep -r "E:ID_PART_ENTRY_NAME=openqapart" /run/udev/data | wc -l) -eq ' . $num_openqapart);
-    assert_script_run('test $(ls -l /run/udev/links/*by-partlabel*{primary,logical}/* | wc -l) -eq 0');
+    script_run('ls -laR '. $udev_label);
+    assert_script_run("test \$(ls -l ${udev_label} | wc -l) -eq 0");
```

Comment:
I'd personally be consistent and either use `'` or `"` (better for readability), but you can ignore it :).

Reply:
I would agree in general but in this particular case double-quoted string is used to be able to replace the variable. Let's keep it 'as is', because otherwise all single-quotes should be replaced with doubled just a sake of unification :)